sql/views/LongRunningQueries.sql

```diff
@@ -25,12 +25,13 @@ outer apply (
         and cs.Setting1 is null
 ) e2
 outer apply (
-    select JobName = concat('Job: ', j.name)
-    from msdb.dbo.sysjobs j
+    select JobName = concat('Job: ', j.name), CategoryName = c.name
+    from msdb.dbo.sysjobs             j
+    inner join msdb.dbo.syscategories c on c.category_id = j.category_id
     where j.job_id = try_convert(uniqueidentifier, try_convert(varbinary(16), (substring(s.program_name, charindex('0x', s.program_name, 0), 34)), 1))
 ) sj
 where r.status in ('running', 'runnable', 'suspended')
-    and (sj.JobName not like '%cdc.%' or sj.JobName is null)
+    and ((sj.JobName not like '%cdc.%' and sj.CategoryName not like 'REPL-%') or sj.JobName is null)

```

Comment:
Should this be `not like 'repl%'` (drop the trailing dash)? A quick check of job categories on a prod server I run (albeit w/o transactional replication) shows the `Replication` category which would not be excluded by this expression) 

<img width="399" alt="image" src="https://user-images.githubusercontent.com/10898767/186935972-8e9cfe01-2b8f-443b-bafc-677b601d114f.png">

Notably CDC jobs are in the categories `REPL-LogReader` and `REPL-Checkup`

Reply:
Hm, I thought 'Replication' was a custom one since none of my replication jobs were in it, but it does seem to be a standard one. I wonder if its a legacy holdover from earlier versions. Either way, happy to make it more broad if that works for you.